querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -176,6 +177,14 @@ export const DataDocScheduleForm: React.FunctionComponent<
                 const notificationField = (
                     <>
                         <FormSectionHeader>Notification</FormSectionHeader>
+                        <div>
+                            Notification will be sent to the owner of the
+                            datadoc{' '}
+                            <b>
+                                @<UserName uid={kwargs.user_id} />
```

Comment:
there is no kwargs.user_id if it is isCreateForm? maybe we can just hide this field